### DISCUSSION TITLE
Missing documentation for Date picker machine

### DISCUSSION THREAD

**Discussion by arekrgw** [chakra-ui/zag#1284] [General]
# 🐛 Bug report

Missing documentation for Datepicker

## 💥 Steps to reproduce

1. Go to docs website
2. Look for datepicker docs
3. There is no entry in menu for that
4. Direct navigation from Ark-ui docs via link shows 404 (https://zagjs.com/components/react/date-picker)

**Comment by segunadebayo** [comment]
Closing — docs have been fully rebuilt for v1.0.
